SPECS/python-coverage/python-coverage.spec

```
@@ -42,7 +42,7 @@ executable, and which have been executed.
 
 %check
 pip3 install tox PyContracts
-LANG=en_US.UTF-8 tox -e py36
+LANG=en_US.UTF-8 tox -e py37
```

I think our python version is `3.9` and maybe this might not be able to find the `py37` env
```suggestion
LANG=en_US.UTF-8 tox -e py39
```